sql.txt

```diff
@@ -1,14 +1,32 @@
 (1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
 Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
 
+SELECT CustomerName, ContactName,Country FROM [Customers] ORDER BY country LIMIT 10 OFFSET 10
+
 (2) Select all columns from the Customer table where the ContactName starts with A.  Paste your SQL statement below.
 
+SELECT * FROM CUSTOMERS WHERE ContactName LIKE "A%"
+
 (3) Select all columns from the OrderDetails table where the ProductID is 51 and the quantity is greater than 10.
 Paste your SQL statement below.
 
+SELECT * FROM [OrderDetails] WHERE ProductID = 51 AND Quantity >10
+
 (4) Insert 3 rows into the Products table.  Note that you will have to specify a valid SupplierID and CategoryID, 
 corresponding to rows from the Supplier and Category tables.  Paste your three SQL statements below.
 
+INSERT INTO [Products] (ProductID, ProductName, SupplierID, CategoryID, Unit, Price)
+	VALUES		(NOT NULL, "ID Product #1",20,5, "10 kg box", 9.99),
+    			(NOT NULL, "ID Product #2",21,5, "5 lb bag", 50),
+    			(NOT NULL, "ID Product #3",22,5, "10 ml", 109.99),
+
+
 (5) Update the two top rows of the Products Table to increase the price by 1.50. (Get SQL to do the addition for you.)  Paste your SQL statement below.
 
+UPDATE [Products] SET Price=Price+1.5 WHERE ProductID=1
+UPDATE [Products] SET Price=Price+1.5 WHERE ProductID=2
```

Comment:
This is great, but you can also use `IN` operator in a single statement like following: 
`UPDATE [Products] SET Price=Price+1.5 WHERE ProductID IN (1, 2)`